File: examples/filter_bilateral_2df_test/filter_bilateral_2df_test.cpp
#include <GL/glew.h>
#include <piccante.hpp>

int main()
{
    
    pic::Image imgIn;
    imgIn.Read("tommaseo_statue.png");

    pic::Image imgOut = pic::FilterBilateral2DF::execute(&imgIn, &imgOut, 4.0f, 0.05f);

    imgOut.Write("filtered.png");
}

File: examples/filter_bilateral_2df_test/filter_bilateral_2df_test.py
import pyccante as py

imgIn = py.Image("pacman.png", py.LDR_type.LT_NOR)
imgOut = py.FilterBilateral2DF.execute(imgIn, 1.5,100.6) 
imgOut.Write("filtered.png", py.LDR_type.LT_NOR)